src/applications/static-pages/sass/modules/_m-education-wizard.scss

```diff
@@ -18,6 +18,24 @@
     background-image: url("/img/arrow-up-white.svg");
   }
 }
+.wizard-button-10203 {
```

Comment:
This should go in https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/sass/edu-benefits.scss

Reply:
Thats were the original css class name was kept but i'll move it over.